isc.class.php

```diff
@@ -216,7 +216,7 @@ public function default_options() {
 			$default['overlay_included_images']   = '';
 			$default['block_options']             = true;
 			$default['enable_log']                = false;
-			$default['standard_source']           = '';
+			$default['standard_source']           = 'custom_text';
```

Comment:
**Suggestion:** Consider using a constant for the string 'custom_text' as it is used multiple times in the code. This will make the code more maintainable and less prone to errors. [maintainability]
```suggestion
   define('DEFAULT_STANDARD_SOURCE', 'custom_text');
   $default['standard_source'] = DEFAULT_STANDARD_SOURCE;
```